waterripples/index.ets

```diff
@@ -0,0 +1 @@
+export { Ripples } from './src/main/ets/components/MainPage/Ripples'
```

Comment:
add copyright text

Reply:
I have now added the copyright text in all the files mentioned